cfg/config.go

```diff
@@ -204,8 +204,15 @@ type SerialTNCConfig struct {
 	// Serial port (e.g. /dev/ttyUSB0 or COM1).
 	Path string `json:"path"`
 
-	// Baudrate for the serial port (e.g. 57600).
-	Baudrate int `json:"baudrate"`
+	// SerialBaud is thee  the serial port's baudrate (e.g. 57600).
+	SerialBaud int `json:"serial_baud"`
+
+	// HBaud is the the packet connection's baudrate (1200 or 9600).
+	HBaud int `json:"hbaud"`
+
+	// Legacy baudrate field (deprecated).
```

Comment:
You can make this stronger by using [GoDoc deprecation](https://go.dev/blog/godoc):

> To signal that an identifier should not be used, add a paragraph to its doc comment that begins with “Deprecated:” followed by some information about the deprecation.

This causes the field to be ~struck out~ in many IDEs. I used this above on `FormsPath`.

Reply:
Nice! That's a neat feature 👍